docs/KNOWN_ISSUES.md

```diff
@@ -49,3 +63,31 @@ If you *must* call `player.reset()` then you'll have to manually restart the `vi
         // Add your plugin configuration here
     });
     ```
+
+## Setting start position on iOS safari
+
+On iOS Safari, there is a compatibility issue with the `loadedmetadata` event when attempting to set the start position
+of a video. This event, commonly used for this purpose on other browsers, may lead to unexpected behavior on iOS
+devices.
```

Comment:
Is the line break intentional?

Reply:
Please refer to https://github.com/SRGSSR/pillarbox-web/pull/173#discussion_r1453592263